tests/test_core.py

```diff
@@ -239,19 +312,28 @@ def _validate_soundscape_and_event_audio(orig_wav_file, gen_wav_file,
                 os.makedirs(orig_events_path)
                 os.makedirs(gen_events_path)
 
-                sc.generate(orig_wav_file.name, orig_jam_file.name,
+                sc.generate(audio_path=orig_wav_file.name,
+                            jams_path=orig_jam_file.name,
+                            txt_path=orig_txt_file.name,
                             disable_instantiation_warnings=True,
                             save_isolated_events=True, 
                             isolated_events_path=orig_events_path)
+
                 scaper.trim(orig_wav_file.name, orig_jam_file.name,
                             orig_wav_file.name, orig_jam_file.name,
                             np.random.uniform(0, 5), np.random.uniform(5, 10))
-                scaper.generate_from_jams(orig_jam_file.name, gen_wav_file.name,
+
+                scaper.generate_from_jams(orig_jam_file.name,
+                                          audio_outfile=gen_wav_file.name,
+                                          txt_path=gen_txt_file.name,
                                           save_isolated_events=True, 
                                           isolated_events_path=gen_events_path)
 
                 _validate_soundscape_and_event_audio(orig_wav_file, gen_wav_file, 
                     gen_events_path, orig_events_path)
+
+                # TODO
```

Comment:
What's this TODO for?

Reply:
Look at the following line, it's commented out but once supported should be uncommented